TestAssemblies/SimpleTest/AsyncClass.cs

```diff
@@ -5,17 +5,17 @@ namespace SimpleTest {
     public class AsyncClass {
         [Interceptor]
         public async Task SimpleAsyncMethod() {
-            await Task.Delay(1);
+            await Task.Delay(1).ConfigureAwait(false);
```

Comment:
I have removed the other two calls to ConfigureAwait() because they are not needed when awaiting completed tasks.